### DISCUSSION TITLE
NVM won't work after installing it multiple times

### DISCUSSION THREAD

**Discussion by kdbbrown** [coreybutler/nvm-windows#1149] [Support] [answered]
I downloaded the latest  nvm release , after installation i get the follozing error 

 the item nvm.exe that is this shortcut refers to has been changedor movef so this shortcut wil not work properly
 
 can someone help me fix it

**Comment by coreybutler** [comment]
> I downloaded the latest  nvm release , after installation i get the follozing error 
> 
>  the item nvm.exe that is this shortcut refers to has been changedor movef so this shortcut wil not work properly
>  
>  can someone help me fix it

Which installer are you using? How are you trying to run nvm? The official installer doesn't create shortcuts,  so I'm not sure how a shortcut was created in the first place.

**Comment by kdbbrown** [comment]
I download this installer [setup.exe](https://github.com/coreybutler/nvm-windows/releases/download/1.1.12/nvm-setup.exe) , 
could it be because i'm trying to install it into my work computer , admin access was needed to install it but it is installed in the admin profile , the shortcut could be as a result of my trying to access it from my profile,

**Comment by kdbbrown** [comment, marked as answer]
@coreybutler  After weeks of trying to make it work i found a way around , 
I was given a computer at work which asks for admin passwords each time i need to install it  , 
Basically they're two profiles - "C:\Users\Admin" and "C:\Users\me" , 
# This was the pathway that worked for me 
 - *Installing nvm on pc with compulsory admin restrictions,* 
- download .exe from repo
- install in Path "C:\Users\<user>\AppData\Roaming\nvm" (require admin password to launch installation)
- set this as symlink "C:\Users\<user>\Roaming\nvm\nodejs"
- update user environment variables NVM_HOME , NVM_SYMLINK 
- add env variables to path
- restart terminal , test with nvm --version (should display nvm version if previous steps are successful)
- install different node version with nvm install <node-verion>
- turn nvm on with nvm on (require admin password)
- switch node version with nvm use <node-version> (require admin password)
Everything should work fine from here , but note each time the node version will need to be changed the admin must be avaibale
